From 2e6b8af52042eadc0ca9f8cac3a8c384671e157b Mon Sep 17 00:00:00 2001
From: Pino Toscano <ptoscano@redhat.com>
Date: Fri, 30 Nov 2018 12:41:03 +0100
Subject: [PATCH 2/2] inspect: handle os-release "opensuse-tumbleweed" as
 opensuse

Followup of commit 70407cd622dda6f088a0876e1e1ae669e9f8a281 for openSUSE
Thumbleweed.

(cherry picked from commit 28bd06227b1633fa08c073fe8dbe65d013d7dc9e)
Signed-off-by: Larry Dewey <ldewey@suse.com>
---
 daemon/inspect_fs_unix.ml | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/daemon/inspect_fs_unix.ml b/daemon/inspect_fs_unix.ml
index c0c0a75a4..b6a9af707 100644
--- a/daemon/inspect_fs_unix.ml
+++ b/daemon/inspect_fs_unix.ml
@@ -142,7 +142,8 @@ and distro_of_os_release_id = function
   | "frugalware" -> Some DISTRO_FRUGALWARE
   | "mageia" -> Some DISTRO_MAGEIA
   | "neokylin" -> Some DISTRO_NEOKYLIN
-  | "opensuse" | "opensuse-leap" -> Some DISTRO_OPENSUSE
+  | "opensuse" -> Some DISTRO_OPENSUSE
+  | s when String.is_prefix s "opensuse-" -> Some DISTRO_OPENSUSE
   | "pld" -> Some DISTRO_PLD_LINUX
   | "rhel" -> Some DISTRO_RHEL
   | "sles" | "sled" -> Some DISTRO_SLES
-- 
2.21.0

